__DEFINES/profile.dm

```diff
@@ -0,0 +1,29 @@
+#define PROFILE_START ;PROFILE_STORE = list();PROFILE_SET;
```

Comment:
These have no reason reason to be defines.

Reply:
Eh, I like them being defines, makes what you're doing more explicit. Can we keep it as is?

Reply:
Fine I guess.